apps/versions/urls.py

```diff
@@ -6,8 +6,10 @@
 
 urlpatterns = patterns(
     '',
-    url('^$', views.version_list, name='addons.versions'),
-    url('^format:rss$', VersionsRss(), name='addons.versions.rss'),
+    url('^(?:status:(?P<status>[^/]+))?$',
```

Comment:
The URL should be `/en-US/firefox/addon/foobar/versions/status:beta` (mentioned in the description of the pull request) or `/en-US/firefox/addon/foobar/versions/status:beta/format:rss`, this is intentional. IMHO, this is consistent with the way parameters are passed on other URLs. What do you think? Removing `status:` is easy of course.